tests/unit/dict_tests.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+
+from wordy.base.worddict import WordDict
+
+
+class TestBoard(unittest.TestCase):
+
+    def test(self):
+        d = WordDict("../assets/dicts/full_dict.txt")
```

Comment:
It might be worth taking out the `../` part of this path, as if we run it in the right working directory it should work just fine. Not a big deal, though, since this is just a test.